src/helix-ui/styles/modules/stepper.less

```diff
@@ -0,0 +1,100 @@
+/* ======================================= *\
+ * BETA BETA BETA BETA BETA BETA BETA BETA *
+\* ======================================= */
+hx-accordion.beta-hxStepper {
+  counter-reset: step;
+
+  hx-accordion-panel {
+    border-top: 1px solid @gray-300;
+
+    &:last-child {
+      border-bottom: 1px solid @gray-300;
+    }
+
+    header[slot="header"] {
+      display: flex;
+      padding: 1rem;
+
+      > * {
+        display: flex;
+        align-items: center;
+      }
+
+      // TODO: clarify mouse hover/click behavior with Design
+      &:hover {
+        .beta-hxStepCounter {
+          background-color: @cyan-500;
+          color: @gray-0;
+        }
+      }
+    }
+
+    &[open] { // NOTE: no spec!
+      .beta-hxStepCounter {
+        background-color: @cyan-500;
+        color: @gray-0;
+      }
+    }
+
+    &[disabled] {
+      .beta-hxStepCounter {
+        background-color: @gray-400;
+        color: @gray-700;
+      }
+
+      .beta-hxStepLabel {
+        color: @gray-700; // NOTE: no spec!
+      }
+
+      header[slot="header"] {
+        &:hover {
+          .beta-hxStepCounter {
+            background-color: @gray-400;
+            color: @gray-700;
+          }
+        }
+      }//header
+    }//[disabled]
+  }//hx-accordion-panel
+
+  .beta-hxStepCounter {
+    background-color: @cyan-900; // to spec
+    border-radius: 3rem; // ensure circle
+    color: @gray-0; // to spec
+    counter-increment: step;
+    flex-shrink: 0;
+    font-size: 1.5rem; // to spec
+    height: 2.25rem; // to spec (36px)
+    position: relative;
+    width: 2.25rem; // 36px, to spec
+
+    &::before {
+      content: counter(step);
+      left: 50%;
+      position: absolute;
+      top: 50%;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  .beta-hxStepLabel {
+    color: @gray-900; // TODO: verify with Design, based on old heading spec
+    flex-basis: 50%; // NOTE: no spec!
+    font-size: 1.5rem; // TODO: verify with Design, based on old heading spec
+    font-weight: 400; // TODO: verify with Design, based on old headin spec
+    margin-left: 1rem; // TODO: verify with Design, doesn't comply with spacing system
```

Comment:
JIRA will be created to update invision specs.

Reply:
JIRA: SURF-954